src/libraries/System.Reflection.Emit.ILGeneration/tests/Label/LabelId.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+
+namespace System.Reflection.Emit.Tests
+{
+    public class LabelId
+    {
+        [Fact]
+        public void LabelId_DefaultConstuctor_ReturnsZero()
+        {
+            Label label1 = new Label();
+            Label label2 = new Label();
+
+            Assert.Equal(0, label1.Id);
+            Assert.Equal(label2.Id, label1.Id);
+        }
+
+        [Fact]
+        public void LabelId_CreatedByILGenerator_ReturnsId()
+        {
+            TypeBuilder type = Helpers.DynamicType(TypeAttributes.NotPublic);
+            MethodBuilder method = type.DefineMethod("Method", MethodAttributes.Public);
+            ILGenerator ilGenerator = method.GetILGenerator();
+            for (int i = 0; i < 100; i++)
+            {
+                Label label = ilGenerator.DefineLabel();
+                Assert.Equal(i, label.GetHashCode());
+            }
+        }
```

Comment:
These seems like unnecessary validation on behavior that the system should be agnostic too. For example, no one should be taking a dependency on `GetHashCode()` being the ID of the label, that seems like a bad assumption or policy to define. The default initialization portion does seems reasonable, but unsure if providing a lot of value.

Reply:
Sorry my bad, the tests copied from [LabelGetHashCode](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Reflection.Emit.ILGeneration/tests/Label/LabelGetHashCode.cs) test and forgot to update the `Assert.Equal(i, label.GetHashCode());` row
```suggestion
        [Fact]
        public void LabelId_DefaultConstuctor_ReturnsZero()
        {
            Label label1 = new Label();
            Label label2 = new Label();

            Assert.Equal(0, label1.Id);
            Assert.Equal(label2.Id, label1.Id);
        }

        [Fact]
        public void LabelId_CreatedByILGenerator_ReturnsId()
        {
            TypeBuilder type = Helpers.DynamicType(TypeAttributes.NotPublic);
            MethodBuilder method = type.DefineMethod("Method", MethodAttributes.Public);
            ILGenerator ilGenerator = method.GetILGenerator();
            for (int i = 0; i < 100; i++)
            {
                Label label = ilGenerator.DefineLabel();
                Assert.Equal(i, label.Id);
            }
        }
```